MAUI/Scheduler/resource-view.md

```diff
@@ -0,0 +1,330 @@
+---
+layout: post
+title: Resource in MAUI Scheduler control | Syncfusion
+description:  Learn here all about grouping the appointments based on the resources in the day, week, and timeline views in Syncfusion WinUI Scheduler (SfScheduler) control.
+platform: maui
+control: SfScheduler
+documentation: ug
+---
+
+# Resource in MAUI Scheduler (SfScheduler)
+
+The scheduler resource view will allow the group appointments based on the resources or dates, arranged by the column, or row in the day, week, workweek, timeline day, timeline week, timeline workweek, and timeline month views. It also allows to share the events or appointments to the multiple resources and resource appointment details that can be edited by using a built-in appointment editor dialog.
```

Comment:
The .NET MAUI Scheduler control allows you to group appointments based on the resources associated with them in the timeline day, timeline week, timeline workweek, and timeline month views, with complete business object binding, multi resource appointment sharing and UI customization features.